src/ray/common/client_connection.h

```diff
@@ -15,24 +15,27 @@
 #ifndef RAY_COMMON_CLIENT_CONNECTION_H
 #define RAY_COMMON_CLIENT_CONNECTION_H
 
-#include <deque>
-#include <memory>
-
-#include <boost/asio.hpp>
+#include <boost/asio/basic_stream_socket.hpp>
+#include <boost/asio/buffer.hpp>
 #include <boost/asio/error.hpp>
+#include <boost/asio/generic/stream_protocol.hpp>
 #include <boost/enable_shared_from_this.hpp>
+#include <deque>
```

Comment:
C++ standard headers should still come first, before the boost ones

Reply:
I'll give it a shot, but I seem to recall the linter complaining when I've done that in the past.

Reply:
Done